docs/REST API/rest-api.md

```diff
@@ -10098,21 +10087,66 @@ Retrieves information about the specified operating system (OS) version.
 }
 ```
 
-OS vulnerability data is currently available for Windows and macOS. For other platforms, `vulnerabilities` will be an empty array:
+Windows and macOS listed vulnerabilities are based on OS version-specific data.
+
+Linux vulnerabilities are based on kernel vulnerabilities for hosts running the specified OS version. Both active and inactive kernels on a host are accounted for in kernel vulnerability reporting, so host counts across kernels may sum to more than the count of hosts running a given OS version. Hosts running in a container or paravirtualized environment do not have a kernel of their own.
 
 ```json
 {
-  "id": 321,
-  "hosts_count": 1,
-  "name": "CentOS Linux 7.9.2009",
-  "name_only": "CentOS",
-  "version": "7.9.2009",
-  "platform": "rhel",
+  "os_version_id": 321,
```

Comment:
Sorry @iansltx 
I think our existing documentation is wrong too.

So what I'm seeing is `os_version_id` is WITHIN `os_version` object. so line 10065 is incorrect. I'm assuming the docs below should match the same pattern as mac/windows having the information in the `os_version` object accordingly?

Reply:
I just pushed fixes <3